### DISCUSSION TITLE
Cycling Failed state on forwarding nodes

### DISCUSSION THREAD

**Discussion by BSUNetSec** [Security-Onion-Solutions/securityonion#11922] [Unsupported Versions] [answered]
Greetings,

Just did the upgrade to 2.3.280 and started having an issue.  I am seeing all of the forwarding nodes cycle to a fault state.  checking the so-status on the forwarding nodes and it looks like the so-zeek is what is cycling a failing state.  running docker logs so-zeek and it returned this.  Not sure what to do next.  We have planned to migrate to 2.4 later this month but would like to keep this one running for the time being.

Thanks

`docker logs so-zeek
checking configurations ...
logger scripts failed.
error in /opt/zeek/share/zeek/policy/securityonion/communityid.zeek, line 24: unknown identifier hash_conn, at or near "hash_conn"

manager scripts failed.
error in /opt/zeek/share/zeek/policy/securityonion/communityid.zeek, line 24: unknown identifier hash_conn, at or near "hash_conn"

proxy scripts failed.
error in /opt/zeek/share/zeek/policy/securityonion/communityid.zeek, line 24: unknown identifier hash_conn, at or near "hash_conn"

worker-1-1 scripts failed.
error in /opt/zeek/share/zeek/policy/securityonion/communityid.zeek, line 24: unknown identifier hash_conn, at or near "hash_conn"

/usr/local/sbin/zeek.sh: line 8: kill: (32) - No such process
checking configurations ...
logger scripts failed.
error in /opt/zeek/share/zeek/policy/securityonion/communityid.zeek, line 24: unknown identifier hash_conn, at or near "hash_conn"

manager scripts failed.
error in /opt/zeek/share/zeek/policy/securityonion/communityid.zeek, line 24: unknown identifier hash_conn, at or near "hash_conn"

proxy scripts failed.
error in /opt/zeek/share/zeek/policy/securityonion/communityid.zeek, line 24: unknown identifier hash_conn, at or near "hash_conn"

worker-1-1 scripts failed.
error in /opt/zeek/share/zeek/policy/securityonion/communityid.zeek, line 24: unknown identifier hash_conn, at or near "hash_conn"

/usr/local/sbin/zeek.sh: line 8: kill: (31) - No such process
checking configurations ...
logger scripts failed.
error in /opt/zeek/share/zeek/policy/securityonion/communityid.zeek, line 24: unknown identifier hash_conn, at or near "hash_conn"

manager scripts failed.
error in /opt/zeek/share/zeek/policy/securityonion/communityid.zeek, line 24: unknown identifier hash_conn, at or near "hash_conn"

proxy scripts failed.
error in /opt/zeek/share/zeek/policy/securityonion/communityid.zeek, line 24: unknown identifier hash_conn, at or near "hash_conn"

worker-1-1 scripts failed.
error in /opt/zeek/share/zeek/policy/securityonion/communityid.zeek, line 24: unknown identifier hash_conn, at or near "hash_conn"

/usr/local/sbin/zeek.sh: line 8: kill: (31) - No such process
checking configurations ...
logger scripts failed.
error in /opt/zeek/share/zeek/policy/securityonion/communityid.zeek, line 24: unknown identifier hash_conn, at or near "hash_conn"

`

**Comment by dougburks** [comment]
What version were you running before you updated to 2.3.280?

Did you have any custom Zeek configuration or scripts?

**Reply by BSUNetSec** [reply]
I believe we were on 2.3.270.  I have not built any custom zeek configs or added any scripts.

Thanks

**Reply by dougburks** [reply]
This has something to do with the fact that CommunityID is now built into Zeek itself, but we updated our default Zeek scripts to reflect that and this was not an issue in our testing.

Did soup complete without errors?

If you run `sudo salt-call state.highstate` on your manager, do you get any errors?

If you run that command on a sensor, do you get any errors?

**Reply by BSUNetSec** [reply]
-----Sensor returned 
Summary for local
--------------
Succeeded: 241 (changed=23)
Failed:      0
--------------
Total states run:     241
Total run time:    42.948 s

------Manager returned
Summary for local
--------------
Succeeded: 690 (changed=22)
Failed:      0
--------------
Total states run:     690
Total run time:   165.747 s

**Reply by dougburks** [reply]
If you run `sudo grep community /opt/so/saltstack/default/pillar/zeek/init.sls` on your manager, what is the output?

**Reply by BSUNetSec** [reply]
- policy/frameworks/notice/community-id
- policy/protocols/conn/community-id-logging

**Reply by dougburks** [reply]
That's the new code which is correct. Seems like there is something on your system that is overriding that.

If you run `sudo grep -r communityid /opt/so` on your manager, what is the output?

**Reply by BSUNetSec** [reply]
only 1 return

/opt/so/saltstack/local/pillar/zeek/init.sls:      - securityonion/communityid

**Reply by dougburks** [reply, marked as answer]
Looks like somebody modified your Zeek config. Please do the following:
```
sudo mv /opt/so/saltstack/local/pillar/zeek/init.sls /root/opt-so-saltstack-local-pillar-zeek-init.sls
sudo salt-call state.highstate
```

Then run the following on a sensor to see if Zeek starts working correctly:
```
sudo salt-call state.highstate
```

**Reply by BSUNetSec** [reply]
Ok that seems to have fixed it. Did a diff on the init.sls in the default and the one moved to root and found 

45,46d44
<       - policy/frameworks/notice/community-id
<       - policy/protocols/conn/community-id-logging
51a50
>       - securityonion/communityid

Thanks for all the help
